### PR TITLE
add Rawsec Cybersecurity Inventory

### PR DESCRIPTION
**Pull request by noraj**
related to #1127

### CODE REVIEW

**Review by sammyhori — CHANGES_REQUESTED** [review]
Hi @noraj, thank you for the contribution!

I'm happy to merge this PR once the minor formatting fix is committed.

The repo is large and the last commit was 2 days ago. My only worry is that that the number of beginner level issues is 3 in 80, with only one of them opened in the last 2 years.

One to keep an eye on... but happy to add for now.

**Comment by noraj** [comment]
> The repo is large and the last commit was 2 days ago. My only worry is that that the number of beginner level issues is 3 in 80, with only one of them opened in the last 2 years.

In fact I wanted to add multiple labels: difficulty::easy, hacktoberfest, type::new content. But it seems the syntax allows only 1 label. Maybe I could just manually batch tag them.

File: data.json [hunk 1]
```diff
@@ -2199,6 +2199,17 @@
...
+            "name": "Rawsec Cybersecurity Inventory",
+            "link": "https://gitlab.com/rawsec/rawsec-cybersecurity-list",
+            "label": "difficulty::easy",
+            "technologies": [
+                "JavaScript",
+                "JSON",
+	        "Pug"
+            ],
+            "description": "An inventory of tools and resources that aims to help people to find everything related to CyberSecurity."
+        },
```
[the remark this answers is not in the document]
Comment (noraj) [reply data.json]: true that JS doesn't care about trailing comma but JSON does.

**Review by sammyhori — APPROVED** [review]

**Comment by sammyhori** [comment]
LGTM now :), thanks a lot for the work you'e put into this entry.

**Pull request merged**